composer.json

```diff
@@ -26,7 +26,7 @@
         "doctrine/instantiator": "^1.3.1 || ^2.0",
         "doctrine/lexer": "^2.0 || ^3.0",
         "jms/metadata": "^2.6",
-        "phpstan/phpdoc-parser": "^1.20"
```

Comment:
I do wonder if we can support both - 1.X & 2.x. Without it people that are using `slevomat/coding-standard` or `rector` will be not able to use new version of serializer. 
At the moment it is a blocker for me as we need to release some bug for the Union types - but I'm fine with releasing it with only 2.x support after releasing bug fix release.

Reply:
that's not possible, as bc break signature, see change https://github.com/schmittjoh/serializer/pull/1572/files#diff-8daba71112686a0487131b9b0f2347a47887f14a6199c712eee72e587e2eba6e

Reply:
In theory we can have a hacky `if(class_exist(ParserConfig::class))`. It will look terrible but most probably it will do the job :)

Reply:
We can do that indeed :)